manim/grpc/update_protos.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-python \
+poetry run python \
```

Comment:
Would suggest reverting this change in this file. I don't think poetry is needed for this.

Reply:
Also possibly move to `scripts` folder?

Reply:
Poetry is needed to access protoc when developing with poetry. I moved the script to scripts/.